setup.py

```diff
@@ -12,11 +12,10 @@
 # Dependencies of GPflow
 requirements = [
     "numpy>=1.10.0",
-    "scipy>=0.18.0",
+    "scipy>=0.18.0,==1.4.1",  # pinned to ==1.4.1 to satisfy tensorflow requirements
```

Comment:
Johnny come lately here but we should remove the `>=0.18.0` constraint if we're going to pin it anyway. (Just to double check, is that pin strictly necessary?)

Reply:
i left it in cos they're pinned for different reasons. i don't know the reason for `>=0.18.0`

Reply:
if I we remove `>=0.18.0`, then down the line remove `==1.4.1` for some reason, we now have no constraint, while what we really want is `>=1.18.0`